newsletter-bundle/contao/modules/ModuleSubscribe.php

```diff
@@ -94,10 +94,19 @@ protected function compile()
 				'name' => 'subscribe_' . $this->id,
 				'label' => $GLOBALS['TL_LANG']['MSC']['securityQuestion'],
 				'inputType' => 'captcha',
-				'eval' => array('mandatory'=>true)
+				'eval' => array('mandatory'=>true, 'required'=>true)
 			);
 
-			$objWidget = new FormCaptcha(FormCaptcha::getAttributesFromDca($arrField, $arrField['name']));
+			/** @var class-string<FormCaptcha> $strClass */
+			$strClass = $GLOBALS['TL_FFL']['captcha'] ?? null;
+
+			// Fallback to default if the class is not defined
+			if (!class_exists($strClass))
+			{
+				$strClass = 'FormCaptcha';
```

Comment:
```suggestion
				$strClass = FormCaptcha::class;
```
Otherwise the fallback will not work because the `Contao\` namespace prefix is missing.

Reply:
It will still work since we are in the `Contao` namespace here, but it wouldn't hurt to change it 👍 . `ModuleRegistration` always had it like this.

Reply:
Changed in b7a79e3.

Reply:
> It will still work since we are in the `Contao` namespace here

I don’t think so: https://3v4l.org/2fR81#v8.4.5

Reply:
Then I just wrongly assumed because it ways always like this in `ModuleRegistration` 😁 (and no one noticed since this code was unlikely to ever execute).

Reply:
It worked in Contao 4 because of https://github.com/contao/contao/blob/98274a6b07f4877dc369ee7975f9c256c54dd576/core-bundle/src/Resources/contao/forms/FormCaptcha.php#L297

Reply:
I know, yes.